impl/src/main/java/org/jboss/forge/roaster/model/ast/AnnotationAccessor.java

```diff
@@ -149,6 +149,29 @@
       return target;
    }
 
+   public void removeAnnotations(final ASTNode body)
+   {
+      removeAnnotation(getModifiers(body));
+
+   }
+
+   public void removeAnnotations(final SingleVariableDeclaration variableDeclaration)
+   {
+      removeAnnotation(variableDeclaration.modifiers());
+   }
+
+   private void removeAnnotation(final List<?> modifiers)
+   {
+      List<?> modifiersCopy = new ArrayList<Object>(modifiers);
+      for (Object object : modifiersCopy)
```

Comment:
Hmmm ok, I didn't know. TBH I searched the better way to do it and didn't find something admissible so I commit this poor code in order to you guid me ^^

In fact, I find something cool in 1.8
modifiers.removeIf(modifier -> modifier instanceof Annotation);